guides/_subsections/create-a-custom-target.md

```diff
@@ -0,0 +1,84 @@
+## [Create a Custom Target](#create-a-custom-target)
+
+By default, Cryostat automatically discovers target JVMs over remote Java Management Extensions (JMX), using various mechanisms (e.g. Kubernetes API, JDP). However, in some cases (e.g. for Kubernetes API, JMX port is not `9091` and port name is not `jfr-jmx`), Cryostat might not see your applications. Fortunately, you can tell Cryostat about them by specifying Custom Targets by filling out the Custom Target form.
+
+<ol>
+  <li>
+    {% include_relative _subsections/common/navigate-to-topology.md %}
+  </li>
+  <li>
+    {% capture open-custom-target-form-text %}
+    <p>
+      Search for Custom Target tab and click <i>Create</i> to open the Custom Target form.
+      <figure>
+        <a href="{{ site.url }}/images/2.3.0/custom-target-1.png" target="_blank">
+          <img src="{{ site.url }}/images/2.3.0/custom-target-1.png">
+        </a>
+        <figcaption>
+          Use the Custom Target form to tell Cryostat about a JVM application that are not automatically discovered.
+        </figcaption>
+      </figure>
+    </p>
+    {% endcapture %}
+    {% include howto_step.html
+      summary="Open the <i>Custom Target form</i>"
+      image-name="2.3.0/topology-7.png"
+      caption="Click on Catalog Icon to open the Topology Entity Catalog."
+      text=open-custom-target-form-text
+    %}
+  </li>
+  <li>
+    {% include howto_step.html
+      summary="Enter Custom Target definition"
+      image-name="2.3.0/custom-target-2.png"
+      caption="Use the form to enter the Custom Target definition."
+      text="
+      The <b>connection URL</b> is required for Cryostat to attempt to find the target. You can optionally provide an  <b>alias</b> or <b>JMX credentials</b> if the target has JMX authentication enabled.
```

Comment:
The special fields should be in title case e.g. Alias, JMX Credentials, Connection URL